### ISSUE TITLE
Support Ctrl+C on ssh client

### ISSUE THREAD

**Issue by zhuangel** [google/gvisor#422]
1. start sshd in container
2. connect to container via ssh
3. more /var/log/bootstrap.log, then input ctrl+c

`gpgv: Signature made Thu Apr 26 23:38:40 2018 UTC
gpgv:                using RSA key 3B4FE6ACC0B21F32
gpgv: Good signature from "Ubuntu Archive Automatic Signing Key (2012) <ftp
master@ubuntu.com>"
gpgv: Signature made Thu Apr 26 23:38:40 2018 UTC
gpgv:                using RSA key 3B4FE6ACC0B21F32
gpgv: Good signature from "Ubuntu Archive Automatic Signing Key (2012) <ftp
master@ubuntu.com>"
dpkg: warning: parsing file '/var/lib/dpkg/status' near line 5 package 'dpk
g':
 missing description
dpkg: warning: parsing file '/var/lib/dpkg/status' near line 5 package 'dpk
g':
 missing architecture
Selecting previously unselected package base-passwd.
(Reading database ... 0 files and directories currently installed.)
Preparing to unpack .../base-passwd_3.5.44_amd64.deb ...
Unpacking base-passwd (3.5.44) ...
dpkg: base-passwd: dependency problems, but configuring anyway as you reque
sted:
 base-passwd depends on libc6 (>= 2.8); however:
  Package libc6 is not installed.
 base-passwd depends on libdebconfclient0 (>= 0.145); however:
  Package libdebconfclient0 is not installed.
--More--(3%)`

This issue related with https://github.com/google/gvisor/issues/421, for Ctrl+C support seems needs support special special character process in file pkg/sentry/fs/tty/line_discipline.go.

**Comment by abliss** [comment]
Duplicate of #154 ?

**Comment by kevinGC** [comment]
I'd like to leave this open as it's own bug, as we do support some job control (#803) but not control characters yet.

**Comment by abliss** [comment]
OK! I'm very interested in control character handling. From reading the code in `line_discipline.go`, it doesn't look too hard to implement. Is this scheduled already internally? If not, would you consider accepting an external PR?

**Comment by kevinGC** [comment]
(Sorry for the late response)

Yes, we love external PRs! This isn't scheduled internally. `line_discipline.go` is probably the right place to add it.

**Comment by 42wim** [comment]
I think this can be closed, fixed by #8562 and #8597
